Patch-Source: https://github.com/ManageIQ/optimist/commit/81f8af6e10c01c6224ad0ea02a8ce7b81623b8e3
--
From 81f8af6e10c01c6224ad0ea02a8ce7b81623b8e3 Mon Sep 17 00:00:00 2001
From: Mamoru TASAKA <mtasaka@fedoraproject.org>
Date: Wed, 27 Sep 2023 21:47:00 +0900
Subject: [PATCH] Test: use recent Minitest style

Current style has long been to use Minitest instead of MiniTest
at least for 3 years, and with minitest 5.19, MiniTest usage support is
hidden unless explicitly setting ENV["MT_COMPAT"].

https://github.com/minitest/minitest/commit/a2c6c18570f6f0a1bf6af70fe3b6d9599a13fdd6

Switch to use Minitest style.
---
 test/optimist/command_line_error_test.rb | 2 +-
 test/optimist/help_needed_test.rb        | 2 +-
 test/optimist/parser_educate_test.rb     | 2 +-
 test/optimist/parser_opt_test.rb         | 2 +-
 test/optimist/parser_parse_test.rb       | 2 +-
 test/optimist/parser_test.rb             | 2 +-
 test/optimist/version_needed_test.rb     | 2 +-
 test/optimist_test.rb                    | 2 +-
 8 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/test/optimist/command_line_error_test.rb b/test/optimist/command_line_error_test.rb
index f29f205..cfebd51 100644
--- a/test/optimist/command_line_error_test.rb
+++ b/test/optimist/command_line_error_test.rb
@@ -1,7 +1,7 @@
 require 'test_helper'
 
 module Optimist
-  class CommandlineErrorTest < ::MiniTest::Test
+  class CommandlineErrorTest < ::Minitest::Test
     def test_class
       assert_kind_of Exception, cle("message")
     end
diff --git a/test/optimist/help_needed_test.rb b/test/optimist/help_needed_test.rb
index f3c94f0..d287e57 100644
--- a/test/optimist/help_needed_test.rb
+++ b/test/optimist/help_needed_test.rb
@@ -1,7 +1,7 @@
 require 'test_helper'
 
 module Optimist
-  class HelpNeededTest < ::MiniTest::Test
+  class HelpNeededTest < ::Minitest::Test
     def test_class
       assert_kind_of Exception, hn("message")
     end
diff --git a/test/optimist/parser_educate_test.rb b/test/optimist/parser_educate_test.rb
index 1e79588..afd4888 100644
--- a/test/optimist/parser_educate_test.rb
+++ b/test/optimist/parser_educate_test.rb
@@ -2,7 +2,7 @@
 require 'test_helper'
 
 module Optimist
-  class ParserEduateTest < ::MiniTest::Test
+  class ParserEduateTest < ::Minitest::Test
     def setup
     end
 
diff --git a/test/optimist/parser_opt_test.rb b/test/optimist/parser_opt_test.rb
index 48d98e0..06a13e1 100644
--- a/test/optimist/parser_opt_test.rb
+++ b/test/optimist/parser_opt_test.rb
@@ -3,7 +3,7 @@
 
 module Optimist
 
-  class ParserOptTest < ::MiniTest::Test
+  class ParserOptTest < ::Minitest::Test
 
     private
 
diff --git a/test/optimist/parser_parse_test.rb b/test/optimist/parser_parse_test.rb
index 79596b5..4cbc279 100644
--- a/test/optimist/parser_parse_test.rb
+++ b/test/optimist/parser_parse_test.rb
@@ -2,7 +2,7 @@
 require 'test_helper'
 
 module Optimist
-  class ParserParseTest < ::MiniTest::Test
+  class ParserParseTest < ::Minitest::Test
 
   # TODO: parse
     # resolve_default_short_options!
diff --git a/test/optimist/parser_test.rb b/test/optimist/parser_test.rb
index 6b3fac3..fbb3b4a 100644
--- a/test/optimist/parser_test.rb
+++ b/test/optimist/parser_test.rb
@@ -3,7 +3,7 @@
 
 module Optimist
 
-class ParserTest < ::MiniTest::Test
+class ParserTest < ::Minitest::Test
   def setup
     @p = Parser.new
   end
diff --git a/test/optimist/version_needed_test.rb b/test/optimist/version_needed_test.rb
index 88651d0..ef0ef39 100644
--- a/test/optimist/version_needed_test.rb
+++ b/test/optimist/version_needed_test.rb
@@ -1,7 +1,7 @@
 require 'test_helper'
 
 module Optimist
-  class VersionNeededTest < ::MiniTest::Test
+  class VersionNeededTest < ::Minitest::Test
     def test_class
       assert_kind_of Exception, vn("message")
     end
diff --git a/test/optimist_test.rb b/test/optimist_test.rb
index 36bef11..770a66b 100644
--- a/test/optimist_test.rb
+++ b/test/optimist_test.rb
@@ -1,6 +1,6 @@
 require 'test_helper'
 
-class OptimistTest < MiniTest::Test
+class OptimistTest < Minitest::Test
   def setup
     Optimist.send(:instance_variable_set, "@last_parser", nil)
   end
